spec/unit/plugins/vmware_spec.rb

```diff
@@ -0,0 +1,67 @@
+#
+# Author:: "Christopher M. Luciano" <cmlucian@us.ibm.com>
+# Copyright (C) 2015 IBM Corp.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+require 'spec_helper'
+
+describe Ohai::System, "plugin vmware" do
+  let (:plugin) do
+    plugin = get_plugin("vmware")
+  end
```

Comment:
Since you're not manipulating the plugin after you fetch it, this can be done inline:
```ruby
let(:plugin) { get_plugin("vmware") }
```